go.mod

```diff
@@ -6,20 +6,21 @@ require (
 	github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd // indirect
 	github.com/davecgh/go-spew v1.1.1
 	github.com/gofrs/uuid v3.2.0+incompatible
-	github.com/golang/protobuf v1.3.5
+	github.com/golang/protobuf v1.4.2
 	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
 	github.com/grpc-ecosystem/grpc-opentracing v0.0.0-20180507213350-8e809c8a8645
 	github.com/networkservicemesh/networkservicemesh/controlplane/api v0.3.0
 	github.com/networkservicemesh/networkservicemesh/pkg v0.3.0
 	github.com/networkservicemesh/networkservicemesh/sdk v0.3.0
+	github.com/onsi/gomega v1.10.3
 	github.com/opentracing/opentracing-go v1.1.0
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.1.0
-	github.com/sirupsen/logrus v1.4.2
-	go.ligato.io/vpp-agent/v3 v3.1.0
-	golang.org/x/sync v0.0.0-20190423024810-112230192c58
-	google.golang.org/grpc v1.27.1
-	gopkg.in/yaml.v2 v2.2.8
+	github.com/sirupsen/logrus v1.6.0
+	go.ligato.io/vpp-agent/v3 v3.2.0
```

Comment:
For PRs that bump ligato import version please call that out in the PR description.  Also, is circleci running against your fork?  I may need to fix the main repo settings to cause PRs to run against the proj level repo